<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientDialog</class>
 <widget class="QDialog" name="ClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>753</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>753</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ClientDialog {
	background-image: url(:/bit3.jpeg);
}</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>135</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>     Client</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>711</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#ClientDialog{ background-image: url(:/images/test_2.png);}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="Page_1">
    <widget class="QPushButton" name="pushButton_Login">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>250</y>
       <width>76</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>261</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_ServerIP">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>21</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>服务器ip：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ServerIPAddr">
          <property name="statusTip">
           <string/>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>xxxx.xxxx.xxxx.xxxx</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_port">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>21</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>端口号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_port">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>1 - 65535</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Page_2">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>240</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit:hover{
border-style:ooutset;
border-radius:10px;
padding:6px;
background-color: rgb(255, 255, 127);
		 
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>330</y>
       <width>121</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{color:rgb(255, 0, 0);
background-color: rgb(170, 255, 0);
}
QPushButton{background-color:rgb(170, 255, 255);
font: 14pt &quot;Ubuntu&quot;;}</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>160</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit:hover{
border-style:ooutset;
border-radius:10px;
padding:6px;
background-color: rgb(255, 255, 127);
		 
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>107</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(239, 41, 41);
font: 14pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>用户名:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/0.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>107</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(239, 41, 41);
font: 14pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>密      码:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>330</y>
       <width>121</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{color:rgb(255, 0, 0);
background-color: rgb(170, 255, 0);
}
QPushButton{background-color:rgb(170, 255, 255);
font: 14pt &quot;Ubuntu&quot;;}</string>
     </property>
     <property name="text">
      <string>注册账号</string>
     </property>
    </widget>
    <zorder>lineEdit_2</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>lineEdit</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>pushButton</zorder>
    <zorder>label</zorder>
   </widget>
   <widget class="QWidget" name="Page_3">
    <widget class="QLabel" name="label_OnlineUsers">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>在线用户：</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>393</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_you_are">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
</string>
          </property>
          <property name="text">
           <string>你已经连接到ip：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ServerIP">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Disconnect">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>断开</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="voiceButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>进入语音</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>320</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>退出语音</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendfileBtn">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>320</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Text">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>358</y>
       <width>60</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>输入消息</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_ChatDisplay">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>401</width>
       <height>261</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>323</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_OnlineUsers">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>256</width>
       <height>271</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PrivateChat">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>320</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>私聊</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="textbold">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>320</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton { background-color: rgb(50, 50, 50,0); border-radius: 3px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_bold.png);}
QToolButton:hover { background-color:rgba(172, 178, 148, 100)}</string>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QToolButton" name="textitalic">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>320</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton { background-color: rgb(50, 50, 50,0); border-radius: 3px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_italic.png);}
QToolButton:hover { background-color:rgba(172, 178, 148, 100)}
</string>
     </property>
     <property name="text">
      <string>I</string>
     </property>
    </widget>
    <widget class="QToolButton" name="textUnderline">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>320</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton { background-color: rgb(50, 50, 50,0); border-radius: 3px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_underline.png);}
QToolButton:hover { background-color:rgba(172, 178, 148, 100)}</string>
     </property>
     <property name="text">
      <string>U</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditWrite">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>360</y>
       <width>531</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>360</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>320</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查看个人信息</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QComboBox" name="skinComboBox">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>20</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    border: 1px solid gray;   /* 边框 */
    border-radius: 3px;   /* 圆角 */
    padding: 1px 18px 1px 3px;   /* 字体填衬 */
}</string>
   </property>
   <item>
    <property name="text">
     <string>北湖锦鲤</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>雪中良乡</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>雪中关村</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>夹带私货</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>盛夏美景</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>某教学楼</string>
    </property>
   </item>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>label_5</zorder>
  <zorder>skinComboBox</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
